src/renderers/native/ReactNativeFiberErrorDialog.js

```diff
@@ -26,11 +26,16 @@ function ReactNativeFiberErrorDialog(capturedError: CapturedError): boolean {
   let errorMessage: string;
   let errorStack: string;
   let errorType: Class<Error>;
+  let framesToPop: any;
 
   // Typically Errors are thrown but eg strings or null can be thrown as well.
   if (error && typeof error === 'object') {
     const {message, name} = error;
 
+    // Util methods like invariant set 'framesToPop' to create a stack trace
+    // that is more relevant and blame-able by tooling.
+    framesToPop = (error: any).framesToPop;
+
     const summary = message ? `${name}: ${message}` : name;
 
     errorMessage = `${summary}\n\nThis error is located at:${componentStack}`;
```

Comment:
Actually I think I was creating a new object to handle the case of a string or null being thrown (instead of an `Error`) but I can just change the code to _only_ create a new `Error` in that case.